wandb/sdk/artifacts/artifact_saver.py

```diff
@@ -94,7 +93,7 @@ def save(
                 base_id,
             )
         finally:
-            self._cleanup_staged_entries()
+            pass
```

Comment:
why do you need it if you just pass?

Reply:
Just for the logs to capture the error. I guess another way would be to `except` all exceptions but i don't think that's a great way. Here, i'm not sure we know what errors to except? Than can be a separate task though.

Reply:
```py
try:
    code()
finally:
    pass
```

is equivalent to

```py
code()
```

Reply:
yes yes - changed that in a follow up PR